### DISCUSSION TITLE
아폴로1100추가 요청

### DISCUSSION THREAD

**Discussion by n0373** [KoreanGRF/KoreanBusSet#26] [건의 | Ideas]
![f9b7d4b37868afb208ab950122e7f048ca5decd45b1ae53b45b3860529e043291c93c70554a13f42504a709feb74d0b661483263a5a5b3b979308eaa72c21957ef018ea837157aff6817854ee03dafa7b98d7b39d5b0aac97e0be7efed83670a](https://user-images.githubusercontent.com/68586069/151941916-cb48ef37-2e9f-4b02-9d82-e4319ea325d0.png)

**Comment by telk5093** [comment]
#28
